packages/l10n/lib/src/l10n/app_localizations_en.dart

```diff
@@ -30,4 +30,8 @@ class AppLocalizationsEn extends AppLocalizations {
 
   @override
   String get layers => 'Layers';
+
+  // added advnaced options
```

Comment:
Is this comment neccesary? Also typos

Reply:
I just added comments for quick understanding. I apologise for the typo. I will remove all comments made by me

Reply:
No need to apologize :) I forget comments in my code all the time haha